### PR TITLE
Deselect organelle after placing a unique organelle

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

This deselects nucleus, binding agents, etc after you placed them. Currently I think it's a bit confusing for new players about how these unique organelles work, the hover hex is still shown and they expect to be able place it again but can't. This to go along with the organelle button getting greyed out should make it much more clearer.

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/microbe_stage/editor/CellEditorComponent.cs [hunk 1]
```diff
@@ -1480,6 +1480,9 @@ private bool IsValidPlacement(OrganelleTemplate organelle)
             (organelle.Definition.RequiresNucleus && !HasNucleus))
             return null;
 
+        if (organelle.Definition.Unique)
+            OnOrganelleToPlaceSelected(null!);
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/CellEditorComponent.cs:1484]: Overriding nullability for this should not be done like this. Maybe you should write a new method called "DeselectOrganelleToPlace"?

Comment (Kasterisk) [reply src/microbe_stage/editor/CellEditorComponent.cs]: Done.

File: src/microbe_stage/editor/CellEditorComponent.cs [hunk 2]
```diff
@@ -1562,11 +1565,21 @@ private void OnOrganelleToPlaceSelected(string organelle)
...
+        UpdateOrganelleButtons(organelle);
+    }
+
+    private void DeselectOrganelleToPlace()
+    {
+        ActiveActionName = null;
+        UpdateOrganelleButtons(null);
+    }
 
+    private void UpdateOrganelleButtons(string? organelle)
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/CellEditorComponent.cs:1577]:
```suggestion
    private void UpdateOrganelleButtons(string? selectedOrganelle)
```

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
Just one tiny nitpick, but otherwise the code looks fine now.

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**